test/unit/analysis/plant_parts_eia_test.py

```diff
@@ -0,0 +1,253 @@
+"""Tests for timeseries anomalies detection and imputation."""
+import pandas as pd
+
+import pudl.analysis.plant_parts_eia
+
+
+def test_plant_part():
```

Comment:
Is there a reason that all of these individual unit tests need to be combined in a single test function? Or can they easily be broken out into several more specific unit tests, which would make it easier to identify exactly what test has failed when something goes wrong. I see `gens_mega` shows up in many of the checks, but that could be pulled out into a `GENS_MEGA` constant that's used in many of the individual tests.